addons/hr_expense/models/hr_expense.py

```diff
@@ -586,7 +591,7 @@ def _default_bank_journal_id(self):
     ], string='Status', index=True, readonly=True, tracking=True, copy=False, default='draft', required=True, help='Expense Report State')
     employee_id = fields.Many2one('hr.employee', string="Employee", required=True, readonly=True, states={'draft': [('readonly', False)]}, default=lambda self: self.env['hr.employee'].search([('user_id', '=', self.env.uid)], limit=1))
     address_id = fields.Many2one('res.partner', string="Employee Home Address")
-    payment_mode = fields.Selection([("own_account", "Employee (to reimburse)"), ("company_account", "Company")], related='expense_line_ids.payment_mode', default='own_account', readonly=True, string="Paid By")
+    payment_mode = fields.Selection(_get_payment_mode_selection, related='expense_line_ids.payment_mode', default='own_account', readonly=True, string="Paid By")
```

Comment:
from V11, you don't need to put the selection in a related field.

good code:
`payment_mode = fields.Selection(related='expense_line_ids.payment_mode', readonly=True, string="Paid By")`

Reply:
Thanks @fmdl